java/number-formatting.md

```diff
@@ -0,0 +1,36 @@
+# NumberFormatting 
+
+## Assignment
+Format a number for a specific locale. For example, the number 12345.124 should be formatted as `12,345.12` in the USA locale, but `12.345,78` in the German locale.
```

Comment:
You were totally right. I just overlooked this & changed the wrong numbers which not only occurred here. Thanks!